router/router.go

```diff
@@ -551,6 +562,7 @@ func (worker *workerT) processDestinationJobs() {
 
 	for _, destinationJob := range worker.destinationJobs {
 		var attemptedToSendTheJob bool
+		var errorAt string
```

Comment:
Maintaining an `errorAt` variable without an error having necessarily happened can be confusing for someone reading the code. Would it be better to rename it to `routerStep`?
Same for the constants, e.g. `ERROR_AT_TF` -> `TRANSFORMATION_STEP`

Reply:
The intention here is to specifically send the step at which error occurred while processing an event in router. Hence the naming was chosen to be `errorAt` as it communicates the same thing. But if it is not intuitive, it can be renamed. Please let me know if you think it is not very intuitive

Reply:
It is a subjective comment, since intuitiveness is not something objective. I am not insisting as long as I am the only one with this consideration :)